static/src/javascripts/projects/common/utils/ajax-promise.js

```diff
@@ -13,6 +15,11 @@ define([
             })
             .fail(function (request, text, err) {
                 var error = err ? err : new Error(text);
+
+                var statusText = (err && err.statusText) || '';
+                var statusCode = (err && err.status) || '';
+                raven.captureException('Error retrieving data (' + text + ') (Status: ' + statusCode + ') (StatusText: ' + statusText + ')');
```

Comment:
I would make this the message, i.e. pass it into `new Error(<msg>)`.

Then, let the consumer decide whether to `raven.captureException`.